code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -280,7 +280,8 @@
 /obj/machinery/capture_the_flag/proc/spawn_team_member(client/new_team_member)
 	var/mob/living/carbon/human/M = new/mob/living/carbon/human(get_turf(src))
 	new_team_member.prefs.copy_to(M)
-	M.set_species(/datum/species/human)
+	if(isplasmaman(M))
+		M.set_species(/datum/species/human) //no plasmeme ctf
```

Comment:
This will not work if future species that do not work are added and will create an obscure thing that would need updating. I think that the species should be forced to human to not allow for species specific benefits / downsides to exist within a gun combat arena however a whitelist could be implemented too (although I think humans only is more elegant and works the best here).

Reply:
I could make a list of allowed species. Also remember its ctf. If your shields are down you're fucked no matter what species you are.

Reply:
I could make a list of allowed species. Also remember its ctf. If your shields are down you're fucked no matter what species you are.

Reply:
> I could make a list of allowed species. Also remember its ctf. If your shields are down you're fucked no matter what species you are.

Yea but different species can have different speeds, abilities (such as dash), movement modes etc.

Reply:
Would the dash really change enough to be considerable? It'll just qdel your own gun if you actually hit someone.  

Movement modes are grod stuff which aren't even in the game yet, but yeah. Whitelist time.